src/ansys/units/dimensions.py

```diff
@@ -122,3 +108,10 @@ def __ne__(self, other):
 
     def __bool__(self):
         return bool(self._dimensions)
+
+
+class IncorrectDimensions(ValueError):
+    """Raised on initialization if a dimension is not of type ``BaseDimensions``."""
```

Comment:
@dnwillia @dnwillia-work I also noticed this wording previously - thanks for correcting it. @hpohekar I believe we might have similar wording in PyFluent(?)